services/web/client/source/class/osparc/data/model/Node.js

```diff
@@ -610,11 +610,15 @@ qx.Class.define("osparc.data.model.Node", {
 
     setOutputData: function(outputs) {
       if (outputs) {
-        for (const outputKey in outputs) {
+        for (const outputKey in this.__outputs) {
           if (!Object.prototype.hasOwnProperty.call(this.__outputs, outputKey)) {
             this.__outputs[outputKey] = {};
           }
-          this.__outputs[outputKey]["value"] = outputs[outputKey];
+          if (!outputs.hasOwnProperty(outputKey)) {
```

Comment:
for the first error you might need to invert the condition? It might not like it written as the negated version